Game Studies.js

```diff
@@ -0,0 +1,119 @@
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+
+
+FW.Scraper({
+  itemType			:'journalArticle',
+  	detect			:FW.Url().match(/gamestudies\.org\/\d+\/articles\/\w+/),
+  	title			:FW.Xpath('//div[@id="article"]/h2').text().trim(),
+  	creators		:FW.Xpath('//div[@id="article"]/small').text().remove(/by/).cleanAuthor("author"),
+  	publicationTitle	:"Game Studies",
+  	abstractNote		:FW.Xpath('//div[@id="article"]/p[1]').text().trim(),
+  	date			:FW.Xpath('//span[@class="date"]').text(),
+  	volume			:FW.Xpath('//span[@class="volume"]').text().match(/volume\s(\d+)/, 1),
+  	issue 			:FW.Xpath('//span[@class="issueno"]').text().match(/issue\s(\d+)/, 1),
+  	tags			:FW.Xpath('//div[@id="article"]/p[2]').text().remove(/^\s*Keywords:\s*/).split(/,/).trim(),
+	attachments:{
+		url			:FW.Url(),
+		title			:"Article Snapshot",
+		type			:"text/html",
+		snapshot		:true
+		},
+  	ISSN			:'1604-7982'
+  });
+
+FW.MultiScraper({
+itemType	:'multiple',
+detect		:FW.Url().match(/gamestudies\.org\/\d+/),
+choices		:{
+	titles	:	FW.Xpath('//a[@class="summary"]').text().trim(),
+	urls	:	FW.Xpath('//a[@class="summary"]').key("href")
+			 }
+});
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://gamestudies.org/1401/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://gamestudies.org/1401/articles/arjoranta",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Game Definitions: A Wittgensteinian Approach",
+				"creators": [
+					{
+						"firstName": "Jonne",
+						"lastName": "Arjoranta",
+						"creatorType": "author"
+					}
+				],
+				"date": "August 2014",
+				"ISSN": "1604-7982",
+				"abstractNote": "Games have been defined and redefined many times over, and there seems to be no end to this continual process or any agreement about the definitions. This article argues that such an agreement is not necessary, and presents a Wittgensteinian approach to discussing game definitions. Instead of the common core approach used in most definitions, this article argues for an approach based on language-games. The common core approach is based on a limited number of shared core attributes, while the language-game approach is based on the idea of family resemblances. The language-game approach sees the cycle of redefinition as a hermeneutic circle that advances our understanding of games. This article also clarifies the distinction between nominal and real definitions and shows how they serve different purposes. With the approach suggested here, the focus in research can shift from the essential attributes of games to understanding definitions as tools for practical purposes.",
+				"issue": "1",
+				"libraryCatalog": "Game Studies",
+				"publicationTitle": "Game Studies",
+				"shortTitle": "Game Definitions",
+				"url": "http://gamestudies.org/1401/articles/arjoranta",
+				"volume": "14",
+				"attachments": [
+					{
+						"title"			:"Article Snapshot",
```

Comment:
clean up spacing. Scaffold should be able to do that for you if you re-save the cases